app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,39 @@
+module API
+  module V1
+    class RentsController < ApplicationController
+      include Wor::Paginate
+
+      before_action :authenticate_user!, :find_user
```

Comment:
this shouldn't  be solved with the access token given at the login  or creating the session? 🤔 i mean instead using this method

Reply:
nop, because the user is not the same of the session, because the user id is passed by URL parms